xml/x509_jarm.xml

```diff
@@ -0,0 +1,96 @@
+  -->
+
+  <fingerprint pattern="^2ad2ad16d2ad2ad00042d42d000000332dc9cd7d90589195193c8bb05d84fa$">
+    <description>Tor relay</description>
+    <example>2ad2ad16d2ad2ad00042d42d000000332dc9cd7d90589195193c8bb05d84fa</example>
+    <param pos="0" name="service.product" value="Tor"/>
+    <param pos="0" name="service.vendor" value="Tor Project"/>
+    <param pos="0" name="service.component.cpe23" value="cpe:/a:torproject:tor:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad0002ad2ad0002ad2ad2ad2ad5bf44aec534289dfa8e33148b66cd6c3$">
+    <description>Synology NAS</description>
+    <example>2ad2ad0002ad2ad0002ad2ad2ad2ad5bf44aec534289dfa8e33148b66cd6c3</example>
+    <param pos="0" name="os.device" value="NAS"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="DSM"/>
+    <param pos="0" name="os.vendor" value="Synology"/>
+    <param pos="0" name="hw.vendor" value="Synology"/>
+    <param pos="0" name="hw.device" value="NAS"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad0002ad2ad00042d42d000000ad9bf51cc3f5a1e29eecb81d0c7b06eb$">
+    <description>Proxmox open-source virtualization platform</description>
+    <example>2ad2ad0002ad2ad00042d42d000000ad9bf51cc3f5a1e29eecb81d0c7b06eb</example>
+    <param pos="0" name="service.vendor" value="proxmox"/>
+    <param pos="0" name="service.product" value="proxmox"/>
+    <param pos="0" name="os.vendor" value="Proxmox"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Proxmox"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad16d2ad2ad22c2ad2ad2ad2ad7e5e7dc6f569c9c16238278a408347ef$">
+    <description>Ubiquiti EdgeRouter</description>
+    <example>2ad2ad16d2ad2ad22c2ad2ad2ad2ad7e5e7dc6f569c9c16238278a408347ef</example>
+    <param pos="0" name="hw.vendor" value="Ubiquiti"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="0" name="os.vendor" value="Ubiquiti"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.device" value="Router"/>
+    <param pos="0" name="os.component.cpe23" value="cpe:/a:ui:er-x-sfp:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^21d14d00021d21d21c21d14d21d21d">
+    <description>UniFi-Video Controller</description>
+    <example>21d14d00021d21d21c21d14d21d21d16e53b2674d38f90c8e876dc8f3c879c</example>
+    <example>21d14d00021d21d21c21d14d21d21d3fe87b802002478c27f1c0da514dbf80</example>
+    <param pos="0" name="hw.vendor" value="Ubiquiti"/>
+    <param pos="0" name="hw.device" value="Video controler"/>
+		<param pos="0" name="hw.certainty" value="0.50"/>
+    <param pos="0" name="os.vendor" value="Ubiquiti"/>
+		<param pos="0" name="os.certainty" value="0.50"/>
+    <param pos="0" name="os.family" value="Linux"/>
+  </fingerprint>
+
+  <fingerprint pattern="^07d14d16d21d21d00042d43d000000aa99ce74e2c6d013c745aa52b5cc042d$">
+    <description>Metasploit listener</description>
```

Comment:
Do we know if this is a Pro listener vs a MSF listener?

Reply:
My bad, it's a simple msf listener.